src/components/snapshot/overview/SnapshotOverviewPanel.tsx

```diff
@@ -150,44 +155,53 @@ function SnapshotOverviewPanel(props: SnapshotOverviewPanelProps) {
             </Typography>
           </Grid>
           <Grid item xs={4}>
-            <Typography variant="h6">
-              DUOS Dataset:
-              <InfoHoverButton infoText={duosInfoButtonText} fieldName="DUOS ID" />
-            </Typography>
-            <Autocomplete
-              data-cy="duos-id-editable-field-view"
-              disabled={pendingSave?.duosDataset}
-              className={classes.duosDropdown}
-              componentsProps={{
-                popper: {
-                  style: { width: '600px' },
-                  placement: 'bottom-start',
-                },
-              }}
-              options={duosDatasets}
-              // Setting to null instead of undefined if unset to make sure that this is always a controlled component
-              value={selectedDuosDataset || null}
-              isOptionEqualToValue={(option, val) =>
-                option?.datasetIdentifier === val?.datasetIdentifier
-              }
-              renderInput={(params: any) => (
-                <TextField
-                  {...params}
-                  inputProps={{
-                    ...params.inputProps,
-                    value: pendingSave?.duosDataset ? 'Saving...' : params.inputProps.value,
+            {_.isEmpty(duosDatasets) &&
+              renderTextFieldValue('DUOS Dataset', snapshot.duosFirecloudGroup?.duosId)}
+            {!_.isEmpty(duosDatasets) && (
+              <>
+                <Typography variant="h6">
+                  DUOS Dataset:
+                  <InfoHoverButton infoText={duosInfoButtonText} fieldName="DUOS ID" />
+                </Typography>
```

Comment:
Should we add some text to the info hover button to explain why the dropdown is not showing up?

Reply:
Ah good point!  I'll add that

Reply:
Should we should show "Duos Users Last Synced" (none) in that case too so that there isn't a space in the middle? I'm also okay with saving that for the bigger UI overhaul that we wanted to do later.

Reply:
We actually show that regardless (only when there is a DUOS dataset linked)